mindone/env/init.py

```diff
@@ -0,0 +1,90 @@
+import logging
+import os
+from typing import Optional, Tuple
+
+import mindspore as ms
+from mindspore.communication import get_group_size, get_rank, init
+
+from ..utils.seed import set_random_seed
+from .parallel_config import ParallelConfig
+
+_logger = logging.getLogger(__name__)
+
+
+def init_train_env(
+    mode: int = ms.GRAPH_MODE,
```

Comment:
Can you also support setting different device_target? such as GPU

Reply:
Done. CPU support is unnecessary, I suppose?

Reply:
Yes, CPU does not need to be supported.